zerver/openapi/zulip.yaml

```diff
@@ -6571,6 +6571,25 @@ paths:
         [organization's configuration](https://zulip.com/help/only-allow-admins-to-add-emoji).
 
         `POST {{ api_url }}/v1/realm/emoji/{emoji_name}`
+      x-parameter-description: |
+        As described above, the image file to upload must be provided in the
+        request's body.
+
+        ## Emoji name
+
+        The emoji name can only contain letters, numbers, dashes, and spaces.
+        Upper and lower case letters are treated the same, and underscores (_)
+        are treated the same as spaces (consistent with how the Zulip UI
+        handles emoji).
```

Comment:
I think this paragraph belongs in the `parameters` description for the `emoji_name` field.

Reply:
Hmm, weirdly enough, the parameters table wasn't rendered for this endpoint in its `.md` file. Moving the description to appropriate place and fixing template.